nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/samediff/SameDiffTrainingTest.java

```diff
@@ -150,7 +150,7 @@ public void irisTrainingEvalTest() {
         SDVariable z0 = in.mmul(w0).add(b0);
         SDVariable a0 = sd.math().tanh(z0);
         SDVariable z1 = a0.mmul(w1).add("prediction", b1);
-        SDVariable a1 = sd.nn().softmax(z1);
+        SDVariable a1 = sd.nn().softmax(z1, 0);
```

Comment:
Fix generation to default arg of -1 is used for softmax, as noted earlier (value of 0 here is wrong anyway - it should be 1 or equivalently -1)